yarn.lock

```diff
@@ -4885,6 +4984,33 @@ vega@^3.0.0-beta.31, vega@^3.0.0-rc5:
     vega-wordcloud "1"
     yargs "4"
 
+vega@^3.0.0-rc6:
```

Comment:
@domoritz  It's weird that yarn adds a new entry of Vega here but still leaves the old rc5 above!